### PR TITLE
feat: migrate deprecated eslint-config-node to eslint-config-n, and u…

### PR DESCRIPTION
**Pull request by Charlie-Crowdstrike**
…pdate references to rulesets

### CODE REVIEW

**Comment by kellyselden** [comment]
You might need to keep both, since eslint-config-crowdstrike has a peer dep on eslint-plugin-node.

**Comment by Charlie-Crowdstrike** [comment]
> You might need to keep both, since eslint-config-crowdstrike has a peer dep on eslint-plugin-node.

`eslint-config-crowdstrike` pulls in `eslint-plugin-node` as a devDep, but not as a peerdep, but i did forget to nuke it as a peerDep in this repo

**Comment by Charlie-Crowdstrike** [comment]
Will also make https://github.com/CrowdStrike/eslint-config-crowdstrike-node/pull/84/files redundant

File: package.json [hunk 1]
```diff
@@ -38,15 +38,14 @@
...
+    "eslint-plugin-n": "^14.0.0",
     "remark-cli": "^12.0.0",
     "remark-preset-lint-crowdstrike": "^4.0.0",
     "renovate-config-standard": "^2.0.0",
     "standard-version": "^9.0.0"
   },
   "peerDependencies": {
     "eslint-config-crowdstrike": ">=1",
-    "eslint-plugin-json-files": ">=0.8",
-    "eslint-plugin-node": ">=10"
```
Comment (kellyselden) [inline package.json:50]: Why wouldn't this be "eslint-plugin-n": ">=14"?

Comment (Charlie-Crowdstrike) [reply package.json:50]: You're 100% right

**Pull request merged**